docs/reference/sql/appendix.md

```diff
@@ -146,4 +146,5 @@ The following table shows all operators in the language.
 | `AND`        | logical AND                    | boolean
 | `OR`         | logical OR                     | boolean
 | `BETWEEN`    | test if value within range     | numeric, string
-| `LIKE`       | match a pattern                | string
\ No newline at end of file
+| `LIKE`       | match a pattern                | string
+| `IN`         | short hand for multiple `OR` expressions | all
```

Comment:
Maybe "abbreviate" instead of "shorthand for"?

Reply:
I don't want to get this out of sync with master, since this is a cherry pick.  Is short hand too informal sounding?

Reply:
Just a personal preference, it's fine as-as!